tasks/build/Dockerfile

```diff
@@ -2,84 +2,95 @@ FROM golang:1.10.1
 
 # Dep
 
-RUN mkdir release
+ENV DEP_VERSION 0.4.1
 
-RUN cd release \
-  && wget https://github.com/golang/dep/releases/download/v0.4.1/dep-linux-amd64
+RUN mkdir release \
+  && cd release \
+  && wget https://github.com/golang/dep/releases/download/v$DEP_VERSION/dep-linux-amd64
 
 RUN curl --location \
-  https://github.com/golang/dep/releases/download/v0.4.1/dep-linux-amd64.sha256 \
-  | sha256sum --check
-
-RUN chmod +x release/dep-linux-amd64
-RUN mv release/dep-linux-amd64 $GOPATH/bin/dep
-
-
-# Cosmos SDK
-
-RUN cd / \
-  && git clone --bare https://github.com/cosmos/cosmos-sdk
-
-ARG SDK_COMMIT
-
-## Linux
-
-ENV GOOS=linux
-ENV GOPATH=/go/$GOOS
-RUN mkdir --parents $GOPATH/src/github.com/cosmos
-
-RUN git --git-dir=/cosmos-sdk.git worktree add \
-  $GOPATH/src/github.com/cosmos/cosmos-sdk $SDK_COMMIT
-
-WORKDIR $GOPATH/src/github.com/cosmos/cosmos-sdk
-RUN make get_vendor_deps
-RUN make install
-
-## macOS
-
-ENV GOOS=darwin
-ENV GOPATH=/go/$GOOS
-RUN mkdir --parents $GOPATH/src/github.com/cosmos
-
-RUN git --git-dir=/cosmos-sdk.git worktree add \
-  $GOPATH/src/github.com/cosmos/cosmos-sdk $SDK_COMMIT
-
-WORKDIR $GOPATH/src/github.com/cosmos/cosmos-sdk
-RUN make get_vendor_deps
-RUN make install
-
-## Windows
-
-ENV GOOS=windows
-ENV GOPATH=/go/$GOOS
-RUN mkdir --parents $GOPATH/src/github.com/cosmos
-
-RUN git --git-dir=/cosmos-sdk.git worktree add \
-  $GOPATH/src/github.com/cosmos/cosmos-sdk $SDK_COMMIT
-
-WORKDIR $GOPATH/src/github.com/cosmos/cosmos-sdk
-RUN make get_vendor_deps
-RUN make install
-
-
-# Voyager
-
-FROM node:9.11.1
+    https://github.com/golang/dep/releases/download/v$DEP_VERSION/dep-linux-amd64.sha256 \
+    | sha256sum --check \
+  && chmod +x release/dep-linux-amd64 \
+  && mv release/dep-linux-amd64 $GOPATH/bin/dep \
+  && rmdir release
+
+# Node.js v9.11.1
+
+RUN apt-get update && \
+  apt-get install --yes xz-utils
+
+# from https://raw.githubusercontent.com/nodejs/docker-node/773be66016fd1ff4a860529a80af61277f75c7e2/9/Dockerfile
+
+RUN groupadd --gid 1000 node \
+  && useradd --uid 1000 --gid node --shell /bin/bash --create-home node
+
+# gpg keys listed at https://github.com/nodejs/node#release-team
+RUN set -ex \
```

Comment:
This is cool, I like how we're verifying node and yarn versions through gpg keys.

Reply:
Thanks, although I can't take credit for it.  That code is copied-and-pasted from the official Node.js Dockerfile.

FWIW I also question how valuable the exercise is given that it's not leveraging PGP's Web of Trust.